### ISSUE TITLE
Set up FPS limit for SDK7 scenes

### ISSUE THREAD

**Issue by menduz** [decentraland/sdk#476]
In the [scene-runtime](https://github.com/decentraland/scene-runtime)

**Comment by nearnshaw** [comment]
@menduz  what does this ticket mean exactly?

Are today systems in the scene not limited to running at 30 FPS? Are they running at whatever the machine is capable of running?

**Comment by menduz** [comment]
Correct. This was superseded by ADR-148 though. We should rename it to `On the renderer, ensure the processing quota is being applied as ADR-148 suggests: Based on distance`
